### PR TITLE
Some final tweaks

### PR DESCRIPTION
**Pull request by rakshith91**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/tests/test_large_block_blob.py [hunk 1]
```diff
@@ -36,7 +36,8 @@
 LARGE_BLOCK_SIZE = 6 * 1024 * 1024
 
 # ------------------------------------------------------------------------------
-
+if platform.python_implementation() == 'PyPy':
```
Comment (annatisch) [inline sdk/storage/azure-storage-blob/tests/test_large_block_blob.py]: I think you need to `import platform`

File: sdk/storage/azure-storage-queue/tests/test_queue.py [hunk 2]
```diff
@@ -813,6 +813,7 @@ def test_unicode_create_queue_unicode_name(self):
 
     @record
     def test_unicode_get_messages_unicode_data(self):
+        pytest.skip("Uncomment after msrest fix")
```
Comment (annatisch) [inline sdk/storage/azure-storage-queue/tests/test_queue.py]: Is it possible that we could make this skip conditional on Python 2.7?

File: sdk/storage/azure-storage-blob/tests/test_large_block_blob.py (hunk 1 above)
Comment (rakshith91) [reply sdk/storage/azure-storage-blob/tests/test_large_block_blob.py]: did that :D

File: sdk/storage/azure-storage-queue/tests/test_queue.py (hunk 2 above)
Comment (kristapratico) [reply sdk/storage/azure-storage-queue/tests/test_queue.py]: We could use pytest "skipif" - https://pytest.readthedocs.io/en/2.8.7/skipping.html

Comment (rakshith91) [reply sdk/storage/azure-storage-queue/tests/test_queue.py]: done

**Pull request merged**